chapter_introduction/index_vn.md

```diff
@@ -1193,7 +1201,13 @@ Usually, this is referred to as *hierarchical classification*.
 One early example is due to [Linnaeus](https://en.wikipedia.org/wiki/Carl_Linnaeus), who organized the animals in a hierarchy.
 -->
 
-*dịch đoạn phía trên*
```

Comment:
Hmm, đoạn phía trên "Hence, the loss ...." bị sót dấu đóng block comment nên bạn dịch sót. Bạn tự thêm một dòng `-->` vào rồi dịch nhé.

Reply:
Ok anh, sẽ bổ sung.

Reply:
Vâng.